tests/roles/manila_adoption/tasks/main.yaml

```diff
@@ -1,55 +1,17 @@
-- name: deploy podified Manila with cephfs backend
-  when: manila_backend == "cephfs"
-  ansible.builtin.shell: |
-    {{ shell_header }}
-    {{ oc_header }}
-    oc patch openstackcontrolplane openstack --type=merge --patch '
-    spec:
-      manila:
-        enabled: true
-        apiOverride:
-          route: {}
-        template:
-          databaseInstance: openstack
-          databaseAccount: manila
-          manilaAPI:
-            customServiceConfig: |
-              [DEFAULT]
-              enabled_share_protocols=cephfs
-            replicas: 1
-            networkAttachments:
-            - internalapi
-            override:
-              service:
-                internal:
-                  metadata:
-                    annotations:
-                      metallb.universe.tf/address-pool: internalapi
-                      metallb.universe.tf/allow-shared-ip: internalapi
-                      metallb.universe.tf/loadBalancerIPs: 172.17.0.80
-                  spec:
-                    type: LoadBalancer
-          manilaScheduler:
-            replicas: 1
-          manilaShares:
-            share1:
-              customServiceConfig: |
-                [DEFAULT]
-                enabled_share_backends=cephfs
-                enabled_share_protocols=cephfs
-                [cephfs]
-                driver_handles_share_servers=False
-                share_backend_name=cephfs
-                share_driver=manila.share.drivers.cephfs.driver.CephFSDriver
-                cephfs_conf_path=/etc/ceph/ceph.conf
-                cephfs_auth_id=openstack
-                cephfs_cluster_name=ceph
-                cephfs_volume_mode=0755
-                cephfs_protocol_helper_type=CEPHFS
-              replicas: 1
-              networkAttachments:
-              - storage
-    '
+- name: Deploy Podified Manila
+  when: manila_backend == "cephfs" or manila_backend == "cephnfs"
+  block:
+    - name: generate CR config based on the selected backend
+      ansible.builtin.template:
+        src: templates/manila_cephfs.yaml.j2
+        dst: /tmp/manila_cephfs.yaml
```

Comment:
@jistr I'm not sure if there's a preference to render the template elsewhere or it's ok to use /tmp/

Reply:
Elsewhere we've used tmp dir relative to the ansible's working directory, which is the playbook directory:

https://github.com/openstack-k8s-operators/data-plane-adoption/blob/631d249e5fe47273abfd8149f9fb4fd625dcd853/tests/roles/mariadb_copy/defaults/main.yaml#L2

So in that example it becomes `<repo-dir>/playbooks/tmp/mariadb` which is slightly easier to work with manually than digging through the content of `/tmp` i think, but it's not a blocker.